.docker/ci-shadow-fixed/Dockerfile

```diff
@@ -1,15 +0,0 @@
-# moveit/moveit:melodic-ci-shadow-fixed
```

Comment:
are you sure they won't have a shadow-fixed in ROS2? if they will, we should keep this in place with a big 

      TODO: convert to ROS2 similar to https://github.com/ros-planning/moveit2/pull/46

Reply:
We can always bring the file back later